PeepsAt/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
```

Comment:
why don't we move to 5?

Reply:
Good question. Since we are still using Java 8, I was avoiding 5, but maybe it's okay to use it? @tiainen thoughts?

Reply:
Java 8 is still supported in Gradle 5, so that should not hold back upgrading to 5.

Reply:
Ok, while 5 works, we'll stay with 4 for now due to deprecation messages.